src/cgame/cg_local.h

```diff
@@ -1761,6 +1761,7 @@ enum altShader_t
 extern  cgs_t               cgs;
 extern  cg_t                cg;
 extern  centity_t           cg_entities[ MAX_GENTITIES ];
+extern unsigned cg_maxEntities;
```

Comment:
How about `cg_numEntities` for more consistency with sgame `num_entities`

Reply:
I was also thinking about `cg_maxEntitiesInPVS` to be more explicit about the purpose, but that can also be `cg_numEntitiesInPVS` and then we iterate with `<=` test instead of `<` and that may fix the variable shadowing.

Reply:
The PVS part is probably not true anyway because of beacons

Reply:
Wait, `num_entities` is not the amount of entities, it is the highest number used by an entity in the previous game frame. If you spawn 40 drills and remove the 39 first ones, that number will be 40 anyway, with the 39 earlier entities being iterated but skipped because invalid (removed).

The code doesn't iterate the amount of entities, but it iterates the entities up to the one with the highest number among previously active entities, so we can iterate removed entities to actually remove them.

So I suggest `cg_highestPreviousEntityNumberInPVS`.

Reply:
I did 'cg_highestActiveEntity'.